system/security/shared/caching.cpp

```diff
@@ -627,7 +627,13 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
                     return true;
                 }
                 else
+                {
                     matchedRes = res;//allowed at this scope, but must also look at child scopes
+                }
+            }
+            else
+            {
+                matchedRes = nullptr;//don't use parent scope permissions for managed child scope, need to get from LDAP
```

Comment:
@RussWhitehead why not break?

Reply:
Because it may find the exact match in a subsequent loop iteration

Reply:
Make sense.